src/app/themes.tsx

```diff
@@ -67,6 +160,12 @@ export const ThemeUpdateContext = createContext((dark: boolean): void => {
  * Makes a styled-components global registry to get server-side inserted CSS
  * Adapted from https://nextjs.org/docs/app/building-your-application/styling/css-in-js#styled-components
  */
+
+interface Props {
+    children: React.ReactElement;
```

Comment:
this was my bad but you don't need to specify a children prop, it's already in props on an FC and can just be used. This also fixes having to have fragments in random places